mkdocs.yml

```diff
@@ -1,18 +1,19 @@
 site_name: Panel Chat Examples
-site_description: Example recipes for Panel ChatInterface.
+site_description: Example recipes for Panel Chat Components
```

Comment:
I think this site if for Panel Chats in general. Not just Panels ChatInterface components.